src/api/trackerTrackedEntities.ts

```diff
@@ -82,6 +82,7 @@ export interface AttributeValue {
 
```

Comment:
There is a `interface AttributeValue` just above which surprised me in the TD meeting, because it looked like the structure of metadata, not of tracker. In fact, it seems unused. It can be deleted.

Reply:
done